fmc/README.md

```diff
@@ -167,63 +182,103 @@ This field provides the Handle of the DV entry that stores the physical address
 
 ### fmc_tci_dv_hdl
 
-This field provides the Handle into the Data Vault where the TCI<sub>FMC</sub> is stored. TCI<sub>FMC</sub> is a SHA-384 Hash of the FMC Module.
+This field provides the Handle into the DCCM datavault where the TCI<sub>FMC</sub> is stored. TCI<sub>FMC</sub> is a SHA-384 Hash of the FMC Module.
 
-### fmc_cdi_kv_hdl
+### fmc_cdi_ecdsa_kv_hdl
 
-This field provides the Handle into the Key Vault where the CDI<sub>FMC</sub> is stored.
+This field provides the Handle into the Key Vault where the ECDSA CDI<sub>FMC</sub> is stored.
 
-### fmc_priv_key_kv_hdl
+### fmc_cdi_mldsa_kv_hdl
 
-This field provides the Handle into the Key Vault where the PrivateKey<sub>FMC</sub> is stored.
+This field provides the Handle into the Key Vault where the MLDSA CDI<sub>FMC</sub> is stored.
 
-### fmc_pub_key_x_dv_hdl, fmc_pub_key_y_dv_hdl
+### fmc_priv_key_ecdsa_kv_hdl
```

Comment:
Just a note, this is technically not needed since FMC can just re-derive the private key from the CDI by passing the same KDF label as ROM did.